cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -264,7 +264,7 @@ var __AdyenHelper : Object = {
                 break;
         }
         paymentMethodObject['encryptedSecurityCode'] = card.adyenEncryptedSecurityCode.value;
-        paymentMethodObject['type'] = "scheme";
+        paymentMethodObject['type'] = card.type.value;
```

Comment:
I would use scheme for new cards and the card type if it is a oneclick\recurring payment. As providign card type for normal cards it will overwrite logic defined in CA. Let's keep CA leading.